Add road runner to the dictionary

ibds-student opened the pull request:
Is this OK?

Commit:
test commit and GitHub actions #5

Commit:
add test for road runner, fails TDD #5

Commit:
add road runner passes test #3

osmart commented on people.py:
```diff
@@ -22,6 +22,14 @@ def _add_oliver(people):
 
+
+def _add_road_runner(people):
+    """ adds oliver """
```
This docstring looks wrong surely
```
""" adds Road Runner """
```

ibds-student replied:
Fixed in 691dc90

osmart commented on people.py:
```diff
@@ -22,6 +22,14 @@ def _add_oliver(people):
+                   age=71, pet=None, hobbies=['running', 'seed'])
+    key = new_p.first_name + '_' + new_p.surname
+    people[key] = new_p
+
```
PEP8 says there should be two blank lines here

ibds-student replied:
fixed in 9d6c0ba 

@osmart is this OK?

Commit:
PEP8 in response to fork request review #6

Commit:
Correct docstring in response to PR review

PR #6

https://github.com/ARU-Bioinf-ISE/forking_exercise/pull/6#discussion_r379839507

osmart commented:
Merge road runner from ibds-student
